htsinfer/rand_nuc_seq.py

```diff
@@ -0,0 +1,249 @@
+from collections import Counter
+
+import numpy as np
+
+def randomize_nucleotide_sequence(input_sequences: list,
+    number_random_seq: int = 1, min_prob: float = 0) -> list:
+    """
+    Returns randomised sequence adhering to the dinucleotide probabilites.
```

Comment:
Please clarify what "dinucleotide probabilities" you are referring to (e.g., "found in a set of input sequences").